gobblin-modules/gobblin-kafka-09/src/test/java/org/apache/gobblin/runtime/KafkaAvroJobStatusMonitorTest.java

```diff
@@ -527,13 +637,14 @@ private GobblinTrackingEvent createGTE(String eventName, Map<String, String> cus
     return new GobblinTrackingEvent(timestamp, namespace, eventName, metadata);
   }
 
-  MockKafkaAvroJobStatusMonitor createMockKafkaAvroJobStatusMonitor(AtomicBoolean shouldThrowFakeExceptionInParseJobStatusToggle, Config additionalConfig) throws IOException, ReflectiveOperationException {
+  MockKafkaAvroJobStatusMonitor createMockKafkaAvroJobStatusMonitor(AtomicBoolean shouldThrowFakeExceptionInParseJobStatusToggle, Config additionalConfig,
+      GaaSObservabilityEventProducer eventProducer) throws IOException, ReflectiveOperationException {
```

Comment:
your choice, but rather than updating so many calls to now take a no-op version, I might overload this method with a two-param form, that merely forwards to the three-param one here, after constructing the no-op instance.

Reply:
Yeah I realized that after, since I already made the changes I think I'll leave this for now but I'll do that if there's any additional configuration needed in future tests for this file